Modules/Core/Common/include/itkCommonEnums.h

```diff
@@ -132,7 +132,7 @@ class CommonEnums
   };
 
   /**
-   * \class CellGeometry
+   * \enum CellGeometry
    * \ingroup ITKCommon
    * Enums used to specify cell type */
   enum class CellGeometry : uint8_t
```

Comment:
Ideally I think Doxygen should be able to automatically "defer" that `CellGeometry` is an enum, just by looking at the code. So then the whole line saying `* \... CellGeometry`  might simply be removed. But that may be beyond the scope of this pull request, so for now replacing `\class` with `\enum` looks fine to me.

Reply:
Indeed. We could do this instead. But it would be a massive change touching many files.

Is there some reason ITK is explicitly using `\class`?  Did someone think it was required?  Is it a stylistic preference to be explicit? Or...?

Reply:
Simply allowing doxygen to infer that the comments are attached to the enum below (as @N-Dekker suggested) seems best to me, as it will also eliminate needing to mess with KWStyle.  What do you think @dzenanz @thewtex ?

Reply:
If it works, that's fine with me.

Reply:
After removing the \enum, the following warning is produced `Modules/Core/Common/include/itkFloatingPointExceptions.h:37: error: comment doesn't have \class`. What are the possible options? Should a modification be done like here [style overwrite rules](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/ITKKWStyleOverwrite.txt)? Or maybe here [this condition](https://github.com/Kitware/KWStyle/blob/6971e89ec4c763601edf1d0ea0d6f71d42fbca5d/kwsCheckComments.cxx#L197) (adding an enum might be more complex)?

Reply:
Anyone have any thoughts on how should we proceed?

Reply:
@andrei-sandor @seanm I think a style overwrite rule is appropriate in this case.

Thank you for your continued contributions :pray:

Reply:
Is there a way to run KWStyle locally on all ITK files, from our local computers?  (So that we don't have to depend on github CI.)

Reply:
Usually `make KWStyle` will work.